services/web/server/src/simcore_service_webserver/garbage_collector/_core_disconnected.py

```diff
@@ -106,6 +106,9 @@ async def remove_disconnected_user_resources(
                     # inform that the project can be closed on the backend side
                     #
                     try:
+                        _logger.warning(
+                            "Closing services for project '%s'", resource_value
+                        )
```

Comment:
I don't get it. Why you can't just change the log level in the right deployment in osparc-ops-deployment-configuration repository?